sourcegraph/sourcegraph
# chore(plg): migrate invoices list to react-query

Migrates invoices list component to react-query. 
Removes `useApiCall` hook as it's no longer used.




## Test plan
- Tested manually
  - run Sourcegraph isntance in dotcom mode locally
  - sign in as team admin and navigate to "cody/subscription/manage" page
  - confirm invoices list is fetched and displayed



## Changelog
